### DISCUSSION TITLE
Error trying to install ScriptKit

### DISCUSSION THREAD

**Discussion by AmodeusR** [johnlindquist/kit#1239] [Errors] [answered]
I'm trying to reinstall Scriptkit after some issues I had with it. When I tried opening it, it would only open a main.log and nothing more, so I tried reinstalling it, but now it opens a weird windows without background or buttons with only text saying it failed to install with _Unexpected ","... is not valid JSON_ error. [This is the text it comes with.](https://codefile.io/f/8p5ZukV9IS). Since it's REALLY big text, I decided to link it to an outside text tool.

**Comment by johnlindquist** [comment, marked as answer]
@AmodeusR It looks like one of the json files your `~/.kit/db/` directory somehow became corrupted.

You can freely delete all of those files.
You can also delete the entire `~/.kit` directory. 

Once you re-open the app, it will re-create the ~/.kit directory from scratch
